spec/models/unaccompanied_minor_output_spec.rb

```diff
@@ -0,0 +1,178 @@
+require "rails_helper"
+FULL_ATTRIBUTES_LIST = %i[
+  reference
+  created_at
+  type
+  version
+  is_under_18
+  is_living_december
+  is_born_after_december
+  is_unaccompanied
+  is_consent
+  is_committed
+  is_permitted
+  have_parental_consent
+  uk_parental_consent_file_type
+  uk_parental_consent_filename
+  uk_parental_consent_saved_filename
+  uk_parental_consent_file_size
+  uk_parental_consent_file_upload_rid
+  uk_parental_consent_file_uploaded_timestamp
+  ukraine_parental_consent_file_type
+  ukraine_parental_consent_filename
+  ukraine_parental_consent_saved_filename
+  ukraine_parental_consent_file_size
+  ukraine_parental_consent_file_upload_rid
+  ukraine_parental_consent_file_uploaded_timestamp
+  minor_date_of_birth
+  minor_date_of_birth_as_string
+  given_name
+  family_name
+  email
+  has_other_names
+  other_given_name
+  other_family_name
+  other_names
+  phone_number
+  identification_type
+  identification_number
+  no_identification_reason
+  nationality
+  has_other_nationalities
+  other_nationality
+  other_nationalities
+  residential_line_1
+  residential_line_2
+  residential_town
+  residential_postcode
+  different_address
+  sponsor_address_line_1
+  sponsor_address_line_2
+  sponsor_address_town
+  sponsor_address_postcode
+  sponsor_date_of_birth
+  sponsor_date_of_birth_as_string
+  privacy_statement_confirm
+  certificate_reference
+  minor_given_name
+  minor_family_name
+  minor_contact_type
+  minor_email
+  minor_phone_number
+  ip_address
+  user_agent
+  started_at
+  sponsor_declaration
+  adult_number
+  minor_contact_details
+  other_adults_address
+  adults_at_address
+].sort
+RSpec.describe UnaccompaniedMinor, type: :model do
+  describe "completed application" do
```

Comment:
```suggestion
  describe "Transferring an application to foundry" do
```

Reply:
This is too specific (eg: what if we move away away from Foundry?)